test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -ex
```

Comment:
This does much more then just test, so I'd rather it be called that accordingly.

Reply:
I consider everything in it part of our "tests" but if you have a better name I'm all ears :)